### PR TITLE
🚀 Remove a few IE fallbacks from module build

### PR DESCRIPTION
**Pull request by alanorozco**
There are likely more IE-only fallbacks hanging around in `src`.

dist/v0.mjs: Δ `-0.43KB`

### CODE REVIEW

**Review by kristoferbaxter — APPROVED** [review]

File: src/service/ie-intrinsic-bug.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ import {transparentPng} from '../utils/img';
  * @package
  */
 export function ieIntrinsicCheckAndFix(win, opt_platform) {
+  if (IS_ESM) {
```
Comment (samouri) [inline src/service/ie-intrinsic-bug.js:30]: This one is already handled. See https://github.com/ampproject/amphtml/blob/eca5174f4c41ef00ceb9ec4a18b3282aec608fc8/src/service/resources-impl.js#L362-L364

File: src/service/ie-media-bug.js [hunk 2]
```diff
@@ -29,6 +29,10 @@ const TAG = 'ie-media-bug';
  * @package
  */
 export function ieMediaCheckAndFix(win, opt_platform) {
```
Comment (samouri) [inline src/service/ie-media-bug.js:31]: I believe this one is also handled a few lines below the intrinsic check in resources-impl.

https://github.com/ampproject/amphtml/blob/eca5174f4c41ef00ceb9ec4a18b3282aec608fc8/src/service/resources-impl.js#L362-L370

File: src/service/ie-intrinsic-bug.js (hunk 1 above)
Comment (alanorozco) [reply src/service/ie-intrinsic-bug.js]: Oops!

**Review by jridgewell — COMMENTED** [review]
Thanks for finding these!

**Pull request merged**